README.md

```diff
@@ -275,4 +285,20 @@ location / {
 	# backend
 	proxy_pass http://127.0.0.1:8000;
 }
-```
\ No newline at end of file
+```
+#### Setting up an SMTP server
+The current code works with SSL-based SMTP server using username and password
+as authentication. Here are instructions for setting up particular SMTP servers:
+
+##### Google SMTP
```

Comment:
We should not recommend Google. Instead, add a blurb about running without mail server and what the ramifications are.

Reply:
I removed the Google SMTP section, and reworded it slightly to make it more general.